src/components/events/events.css

```
@@ -0,0 +1,48 @@
+.eventcard-grid {
+  display: grid;
+  grid-auto-flow: column;
+  border-radius: 10pt;
+  background-color: var(--events-bg-color);
+  width: 364px;
+  height: 98px;
+}
+
+.eventcard-left {
+  display: grid;
+  grid-auto-flow: row;
+  justify-content: start;
+  align-content: center;
+  padding-left: 20px;
+}
```

Cant see any changes, when justify-content is deleted ;-)

```suggestion
.eventcard-left {
  display: grid;
  grid-auto-flow: row;
  
  align-content: center;
  padding-left: 20px;
}
```

---

True, I think thats the default value, so no ned to specify it. Thanks!